Watchman.Discord/Areas/Responses/Services/ResponsesGetterService.cs

```diff
@@ -25,14 +29,39 @@ public IEnumerable<Response> GetResponsesFromBase()
 
         public IEnumerable<Response> GetResponsesFromResources()
         {
+            Log.Information("Initializing default responses...");
+            var managerMethods = typeof(ResponsesManager).GetMethods();
             var defaultResponses = typeof(Devscord.DiscordFramework.Framework.Commands.Responses.Resources.Responses).GetProperties()
                 .Where(x => x.PropertyType.Name == "String")
                 .Select(prop =>
                 {
                     var onEvent = prop.Name;
                     var message = prop.GetValue(prop)?.ToString();
-                    return new Response(onEvent, message, DEFAULT_SERVER_ID);
+                    var parameters = managerMethods.First(info => info.Name == onEvent).GetParameters().Skip(1);
+                    var availableVariables = new List<string>();
```

Comment:
a co jeśli będzie Contexts a po nim UserContext?

zrób jako HashSet a nie List

Reply:
Po co UserContext po Contexts skoro Contexts zawiera w sobie UserContext

Reply:
żeby nadpisać innym użytkownikiem niż ten, który wysłał wiadomość

Reply:
w sensie availableVariables jako HashSet?

Reply:
tak
no chyba że wrzucić to do osobnej metody która zwraca ienumerable
i yeldować wyniki
a na koniec Distinct

HashSet pod spodem sprawdza czy nie ma duplikatów, ale dodatkowo wszystko hashuje

Reply:
po co tak kombinować z tym Distinct skoro można po prostu zmienić typ kolekcji, co też uczynię